src/index.ts

```diff
@@ -102,17 +99,15 @@ async function main() {
     // Can also use decoder.decodeLogsSync if it is more convenient.
     const decodedLogs = await decoder.decodeLogs(res.data.logs);
 
+    assert.equal(decodedLogs.length, res.data.logs.length);
+
     for (let i = 0; i < decodedLogs.length; i++) {
       const log = decodedLogs[i];
       const rawLogData = res.data.logs[i];
 
       // skip invalid logs
       if (
-        log == undefined ||
-        log.indexed.length < 2 ||
-        log.body.length < 1 ||
-        rawLogData == undefined ||
-        rawLogData.address == undefined
+        log === null || rawLogData.address === undefined || log === undefined
```

Comment:
```suggestion
        log == undefined || rawLogData.address === undefined
```

This should also work afaik.